### DISCUSSION TITLE
Support for arbitary values in tailwind.config.js

### DISCUSSION THREAD

**Discussion by jenstornell** [tailwindlabs/tailwindcss#8069] [Ideas]
When having an arbitary values from the html or a snippet on the web, we might want to move them to the tailwind.config.js file. However the format differs. Arbitary values uses underscores while settings uses css syntax.

Now it's not possible to just copy and paste the arbitary value into the config file which results in some additional manual work.

**Does not work**

```js
module.exports = {
  theme: {
    extend: {
      boxShadow: {
        custom: "0_8px_30px_rgb(0,0,0,0.06)",
      },
    },
  },
};
```

**Works**

```js
module.exports = {
  theme: {
    extend: {
      boxShadow: {
        custom: "0 8px 30px rgb(0,0,0,0.06)",
      },
    },
  },
};
```

### Suggestion

When parsing the values in the settings file, just replace underscores with an empty space and it would probably solve the problem.

**Comment by cannap** [comment]
just write a function 



```
function replace(str) {
return str.replaceAll('_', '  ')
} 

module.exports = {
  theme: {
    extend: {
      boxShadow: {
        custom: replace('0_8px_30px_rgb(0,0,0,0.06)')
      },
    },
  },
};
```
